setup.py

```diff
@@ -216,8 +216,8 @@ def get_extensions():
 Numpy is a dependency for both.
 
 The documentation can be found
-`here <http://hgomersall.github.com/pyFFTW/>`_, and the source
-is on `github <https://github.com/hgomersall/pyFFTW>`_.
+`here <http://pyFFTW.github.io/pyFFTW/>`_, and the source
+is on `github <https://github.com/pyFFTW/pyFFTW>`_.
 '''
```

Comment:
Well if you go the GitHub pages route, it might be worth getting it building on CI. Would recommend going with a dedicated tool instead of writing it out by hand like I did. Maybe something like [this]( https://github.com/Syntaf/travis-sphinx ).